community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/Transaction.java

```diff
@@ -19,7 +19,25 @@
  */
 package org.neo4j.internal.kernel.api;
 
-public interface Transaction extends Read, Write
+public interface Transaction
 {
+    long READ_ONLY_TRANSACTION = 0;
 
+    long commit();
```

Comment:
I _guess_ the long returned here is either the transaction id, or the `READ_ONLY_TRANSACTION` constant?